packages/modules/devices/solax/solax/config.py

```diff
@@ -1,14 +1,19 @@
 from typing import Optional
 
 from modules.common.component_setup import ComponentSetup
+from modules.devices.solax.solax.version import SolaxVersion
 from ..vendor import vendor_descriptor
 
 
 class SolaxConfiguration:
-    def __init__(self, modbus_id: int = 1, ip_address: Optional[str] = None, port: int = 502):
+    def __init__(self, modbus_id: int = 1,
+                 ip_address: Optional[str] = None,
+                 port: int = 502,
+                 version: SolaxVersion = SolaxVersion.G3):
         self.modbus_id = modbus_id
         self.ip_address = ip_address
         self.port = port
+        self.version = version
```

Comment:
Muss die neue Einstellung version nicht auch noch in update_config hinzugefügt werden?